librz/core/cmd/cmd_zign.c

```diff
@@ -557,7 +558,7 @@ static int cmdFlirt(void *data, const char *input) {
 		RzListIter *iter;
 		RzList *files = rz_file_globsearch(input + 2, depth);
 		rz_list_foreach (files, iter, file) {
-			rz_sign_flirt_scan(core->analysis, file);
+			rz_flirt_apply_signatures(core->analysis, file);
```

Comment:
Rename to `rz_sign_flirt_apply()` or something similar.

Reply:
why? i think the new name is more clear.